int16 motorRight
int16 motorLeft
